README.md

```diff
@@ -1,72 +1,53 @@
-
-You can pick the sample that best fits the scenario you are interested in. The instructions for each sample describe how to target [Windows](http://aka.ms/windowscontainers) or Linux Docker images, from Windows, Linux or macOS.
-
-The samples use .NET Core 2.0. They use Docker [multi-stage build](https://github.com/dotnet/announcements/issues/18) and [multi-arch tags](https://github.com/dotnet/announcements/issues/14) where appropriate.
-
-You need to have the [.NET Core SDK](https://www.microsoft.com/net/download/core#/sdk) and [Git](https://git-scm.com/downloads) and [Docker client 17.06 or newer](https://www.docker.com/products/docker) clients installed to use these samples.
+You can pick the sample that best fits the scenario you are interested in, on Windows, macOS or Linux.
 
 ## Getting Started
 
-You can run a [sample application](https://hub.docker.com/r/microsoft/dotnet-samples/) that runs from a pre-built image that has been published to Docker Hub. The source of this sample application is the [dotnetapp-prod](dotnetapp-prod) sample.
+You can run a pre-built [.NET Core sample](https://hub.docker.com/r/microsoft/dotnet-samples/) using [Docker](https://www.docker.com/products/docker). The source of this sample application is the [dotnetapp-prod](dotnetapp-prod) sample.
 
-To run the **Linux** image:
+Type the following command to run the sample:
 
 ```console
 docker run microsoft/dotnet-samples
 ```
 
-To run the **Windows** image:
-
-```console
-docker run microsoft/dotnet-samples:dotnetapp-nanoserver
-```
-
-It is recomended to run the sample twice. The second run will not include downloading the image, so is more representative of typical Docker use.
-
 ## Samples
 
-The following samples show different ways to use .NET Core images.
+The following samples show different ways to use .NET Core images. They depend on the [.NET Core 2.0 Docker images](https://hub.docker.com/r/microsoft/dotnet/) on Docker Hub, provided by the .NET Team at Microsoft. They use Docker [multi-stage build](https://github.com/dotnet/announcements/issues/18) and [multi-arch tags](https://github.com/dotnet/announcements/issues/14) where appropriate.
 
-### Development
+### Building .NET Core Apps with Docker
 
-* [.NET Core Development Sample](dotnetapp-dev) - This sample is good for development and building since it relies on the .NET Core SDK image. It performs `dotnet` commands on your behalf, reducing the time it takes to create Docker images (assuming you make changes and then test them in a container, iteratively).
+* [.NET Core "Starter" Sample](dotnetapp-prod) - This sample builds and runs a single project in Docker. It's a simple sample to start with.
+* [.NET Core End-to-End Sample](dotnetapp-dev) - This sample builds, tests and runs the sample using Docker. It includes and builds multiple projects. 
+* [ASP.NET Core Docker Sample](aspnetapp) - This samples demonstrates a Dockerized ASP.NET Core Web App.
```

Comment:
Nit - `Docker` isn't consistently used in the titles.  `.NET Core Docker Alpine Production Sample` vs `.NET Core End-to-End Sample`